ONSdigital/rm-sample-service
# Migrate to eclipse temurin 17 image

# What and why?
This PR updates the docker image from jdk-17-slim openjdk:17-slim to eclipse-temurin:17-jre-alpine. The package management command has also been updated to work with the new image. 
# How to test?
- Run tests (local and acceptance)
- check image size is smaller than previous image.
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-968
